packages/formation-react/.gitignore

```diff
@@ -1 +1,2 @@
 yarn.lock
+*.js
```

Comment:
Does this capture the `src/helpers` as well? Those are `.js` files.

Reply:
Maybe it should be something like `/*.js`?

Reply:
Ooh, good thinking. I just verified this only captures root-level `.js` files by making a change to `helpers/fields.js` and it looks like it worked as expected; the change was tracked.
![image](https://user-images.githubusercontent.com/12970166/98711246-9d0e4c00-2339-11eb-9944-3d5a8fb03d23.png)

However, _new_ `.js` files didn't get tracked, so changing it to `/*.js` is the way to go. Thanks! :raised_hands: